### PR TITLE
Django 3.0 Py 3.8 support proposal

### PR DESCRIPTION
**Pull request by AndreasDickow**

### CODE REVIEW

File: tests/settings.py [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
-
+import django
 
 urlpatterns = []
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': ':memory:'
+if django.VERSION[:2] >= (3, 0):
```
Comment (yakky) [inline tests/settings.py]: why is this needed?

Comment (AndreasDickow) [reply tests/settings.py]: https://docs.djangoproject.com/en/3.0/releases/3.0/#features-removed-in-3-0
django.db.backends part is removed

Comment (yakky) [reply tests/settings.py]: I only see `The django.db.backends.postgresql_psycopg2 module is removed.`. they removed  in favor of `postgresql`

**Comment by yakky** [comment]
Thanks, I merged all the Django 3 contributions into #50 to ensure a smoother merge process

**Comment by yakky** [comment]
@mbi thanks for your contributiion. Django 3 support has been merged with #50

**Comment by yakky** [comment]
@AndreasDickow  thanks for your contribution. Django 3 support has been merged with #50

**Comment by AndreasDickow** [comment]
@yakky no problem, I'm glad to contribute
